src/Tooltip/tooltip.jsx

```diff
@@ -6,8 +6,13 @@ export function PSATooltip({
   title, 
   placement = 'bottom', 
   arrow = true,
-  enterTouchDelay = 0,  
+  enterTouchDelay = 0, 
+  leaveTouchDelay = 0, 
   tooltipContent, 
+  open,
+  onMouseEnter,
+  onMouseLeave,
+  componentsProps = {},
   ...props
```

Comment:
Please check [MUI documentation for Tooltip](https://mui.com/material-ui/api/tooltip/) for props of `<Tooltip />`, all the props here except for `tooltipContent` and `...props` are listed. In this way I think we can remove them from this component, and when we can directly set them when needed since all of them will be passed by `...props`.